client/src/components/categorical/value/index.tsx

```diff
@@ -69,21 +71,27 @@ type CategoryValueProps = PureCategoryValueProps & {
 // @ts-expect-error ts-migrate(1238) FIXME: Unable to resolve signature of class decorator whe... Remove this comment to see the full error message
 @connect(((state: RootState, ownProps: PureCategoryValueProps) => {
   const { pointDilation, categoricalSelection } = state;
-  const { metadataField, categorySummary, categoryIndex } = ownProps;
+  const { metadataField, categorySummary, categoryIndex, categoryData } =
+    ownProps;
   const isDilated =
     pointDilation.metadataField === metadataField &&
     pointDilation.categoryField === _currentLabelAsString(ownProps);
 
   const category = categoricalSelection[metadataField];
   const label = categorySummary.categoryValues[categoryIndex];
-  const isSelected = category.get(label) ?? true;
+  const col = categoryData.icol(0);
+  const labelName = isDataframeDictEncodedColumn(col)
+    ? col.codeMapping[parseInt(label, 10)]
```

Comment:
Reading dataframe.ts, it looks like any dict-encoded array will already have a fully flattened `values` Array containing this information.    Why do the lookup via `codeMapping` when you could use `values`?   Or perhaps, why create `values` and not use it?

Reply:
I am not sure why I created `values` even though I had no intention to use it. I think I was still uncertain at that point about what I wanted the `DataframeDictEncodedColumn` to look like. I think we would prefer to use int typed arrays everywhere in the frontend and map the integers to string values. I will remove `values`.